moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -241,6 +247,19 @@ void ServoCalcs::start()
   // Stop the thread if we are currently running
   stop();
 
+  // Check if a realtime kernel is installed. Set a higher thread priority, if so
+  if (controller_manager::has_realtime_kernel())
+  {
+    if (!controller_manager::configure_sched_fifo(THREAD_PRIORITY))
```

Comment:
Shouldn't setting the thread priority happen inside the thread where all the servo calculations are happening (i.e. [here](https://github.com/ros-planning/moveit2/blob/13e8bc31b4e4495d372ab28c26f38038592df702/moveit_ros/moveit_servo/src/servo_calcs.cpp#L300))? Currently this would set the priority for whatever thread called `ServoCalcs::start`

Reply:
Good catch, it should be done there. Please update the PR to change which thread is being configured to be the one that runs the main control loop of servo.